src/15_global_state.md

```diff
@@ -180,10 +194,6 @@ data flow and of fine-grained reactive updates.
 
 > **Note**: There are some significant drawbacks to this approach. Both signals and memos need to own their values, so a memo will need to clone the field’s value on every change. The most natural way to manage state in a framework like Leptos is always to provide signals that are as locally-scoped and fine-grained as they can be, not to hoist everything up into global state. But when you _do_ need some kind of global state, `create_slice` can be a useful tool.
 
-[Click to open CodeSandbox.](https://codesandbox.io/p/sandbox/15-global-state-0-5-8c2ff6?file=%2Fsrc%2Fmain.rs%3A1%2C2)
-
-<iframe src="https://codesandbox.io/p/sandbox/15-global-state-0-5-8c2ff6?file=%2Fsrc%2Fmain.rs%3A1%2C2" width="100%" height="1000px" style="max-height: 100vh"></iframe>
-
 <details>
 <summary>CodeSandbox Source</summary>
```

Comment:
It was driven by the idea of bringing examples to the top of the page. Now I'm not sure. 

Maybe, it is fine to keep them to the end of chapter, except the first one. How do you think?

Reply:
I'm fine with the examples in either place, but if we move them I think it would make sense to keep the non-live code next to the live example. That was my main concern because it's a fallback for when code sandbox is down and someone might forget to update them if it isn't next to each other.

Personally I would have kept them in the same place just so I don't have to review all the content to make sure the examples all make sense not being next to the text they were originally next to. I haven't looked at each though so if you think it works that's fine with me.

Reply:
I think, It should look good now.

Thank you!